DOC-426 Fix cloud account creation script for Could API

TobyHFerguson opened the pull request:
Corrected invalid scripting that used `||` as an 'or', which doesn't work for bash!

Commit:
update 50-create-cloud-account.sh

Corrected invalid scripting that used `||` as an 'or', which doesn't work for bash!

banker commented:
@TobyHFerguson https://tldp.org/LDP/abs/html/comparison-ops.html According to these docs, you use -o or logical or, not -a.

banker commented:
Actually @TobyHFerguson: do you have any opposition to replacing this Bash script with a Python script? I think it'd be better for us to provide an example that's less brittle.

banker reviewed (changes requested):
Should we remove this bash script and replace with something in Python?

TobyHFerguson commented:
I'd MUCH prefer this to be python than bash!

*Toby H Ferguson*

Cloud Solutions Architect

Phone: +1 408 386 9343

[image: Toby Ferguson LinkedIn]
<https://www.linkedin.com/in/tobyhferguson> [image:
AWS Certified DevOps Engineer – Professional]



On Tue, Dec 22, 2020 at 6:25 AM Kyle Banker <notifications@github.com>
wrote:

> *@banker* requested changes on this pull request.
>
> Should we remove this bash script and replace with something in Python?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/RedisLabs/redislabs-docs/pull/1156#pullrequestreview-557115439>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABRZCBJ2IU3IDLN5M74EJB3SWCT5PANCNFSM4VE7EIMA>
> .
>

Disclaimer

The information contained in this communication from the sender is confidential. It is intended solely for use by the recipient and others authorized to receive it. If you are not the recipient, you are hereby notified that any disclosure, copying, distribution or taking action in relation of the contents of this information is strictly prohibited and may be unlawful.

This email has been scanned for viruses and malware, and may have been automatically archived by Mimecast Ltd, an innovator in Software as a Service (SaaS) for business. Providing a safer and more useful place for your human generated data. Specializing in; Security, archiving and compliance. To find out more visit the Mimecast website.

Commit:
update 10-create-subscription.sh (#1159)

Updated script because it use '||' as an 'or', which doesn't work for test conditions

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/RedisLabs/redislabs-docs?pullRequest=1156) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/RedisLabs/redislabs-docs?pullRequest=1156) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/RedisLabs/redislabs-docs?pullRequest=1156) it.</sub>
